.gitignore

```diff
@@ -18,3 +18,4 @@ WebOb.egg-info/
 pytest*.xml
 coverage*.xml
 .pytest_cache/
+
```

Comment:
Please don't update the `.gitignore`, I would ask you to please rebase your changes to avoid touching this file at all.